### PR TITLE
Vai toggle extension.

### PR DESCRIPTION
**Pull request by igomez06**
Issue: https://github.com/rancher/qa-tasks/issues/1366

### CODE REVIEW

**Review by floatingman — DISMISSED** [review]
LGTM

**Review by brudnak — APPROVED** [review]
LGTM :shipit:

File: extensions/vai/vai.go [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+		return fmt.Errorf("error when restarting pods")
+	}
+
+	adminClient.Session.RegisterCleanupFunc(func() error {
+		_, err := features.UpdateFeatureFlag(adminClient.Steve, featureFlagResp, false)
+		if err != nil {
+			return err
+		}
+		return nil
+	})
```
Comment (caliskanugur) [inline extensions/vai/vai.go:50]: Is this from the dev stage, it seems like the first iteration of this: https://github.com/rancher/shepherd/pull/218/files#diff-92566b04396c1f7c9b8e22aca23df2d6ffdc48fd4630c1115fbab240f2b1bcbaR87-R93

**Review by caliskanugur — COMMENTED** [review]
Looks pretty solid, I've only one question

File: extensions/vai/vai.go (hunk 1 above)
Comment (igomez06) [reply extensions/vai/vai.go]: Removed it and tested it out.

**Review by floatingman — APPROVED** [review]

**Pull request merged**